packages/@okta/vuepress-site/docs/reference/api/asa/attributes/index.md

```diff
@@ -0,0 +1,288 @@
+
+#### Response Body
+
+On returning a 200: List of attributes
+
+Returns a list of [TeamGroupAttribute](/docs/asa/models.html#teamgroupattribute) objects.
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Authorization: Bearer ${jwt}" \
+https://app.scaleft.com/v1/teams/{team_name}/groups/{group_name}/attributes
+```
+
+##### Response
+```json
+{"list":[{"attribute_name":"unix_group_name","attribute_value":"group_old","id":"dc80df68-ad9f-4294-9a32-9403bbd45a00","managed":false},{"attribute_name":"windows_group_name","attribute_value":"group_new","id":"431418f6-f001-47cd-825f-022e287e1167","managed":false}]}
+```
+### Fetch group attribute
+
+<ApiOperation method="GET" url="/v1/teams/{team_name}/groups/{group_name}/attributes/{attribute_id}" />
+
+
+#### Request Path Parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| team_name   | string |  |
+| group_name   | string |  |
+| attribute_id   | string |  |
+
+
+#### Request Query Parameters
+
+This endpoint has no query parameters.
+
+#### Request Body
+
+This endpoint has no request body.
+
+#### Response Body
+
+On returning a 200: The attribute that was requested
+
+Returns a [TeamGroupAttribute](/docs/asa/models.html#teamgroupattribute) object.
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Authorization: Bearer ${jwt}" \
+https://app.scaleft.com/v1/teams/{team_name}/groups/{group_name}/attributes/{attribute_id}
+```
+
+##### Response
+```json
+{"attribute_name":"unix_group_name","attribute_value":"group_old","id":"dc80df68-ad9f-4294-9a32-9403bbd45a00","managed":false}
+```
+### Update group attribute
+
+<ApiOperation method="PUT" url="/v1/teams/{team_name}/groups/{group_name}/attributes/{attribute_id}" />
+
+
+#### Request Path Parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| team_name   | string |  |
+| group_name   | string |  |
+| attribute_id   | string |  |
+
+
+#### Request Query Parameters
+
+This endpoint has no query parameters.
+
+#### Request Body
+
+*Required:* Attribute to update
+Uses a [UpdateGroupAttribute](/docs/asa/models.html#updategroupattribute) object.
+
+#### Response Body
+
+On returning a 204: The group attribute was updated successfully.
+
+
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X PUT \
+-H "Authorization: Bearer ${jwt}" \
+https://app.scaleft.com/v1/teams/{team_name}/groups/{group_name}/attributes/{attribute_id}
+```
+
+##### Response
+```json
+
+```
+### Lists the attributes for a user
+
+<ApiOperation method="GET" url="/v1/teams/{team_name}/users/{user_name}/attributes" />
+
+
+#### Request Path Parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| team_name   | string |  |
+| user_name   | string |  |
+
+
+#### Request Query Parameters
+
+%List any query parameters here in alpha order%
+
+| Parameter | Description   | Required |
```

Comment:
Column headings should be `Parameter | Type | Description`

Reply:
Information about whether a parameter/property is required or not should be in the description. Specifically, there should be a parenthetical `(Optional)` at the beginning of the description.